src/app/home/home.component.html

```diff
@@ -1,18 +1,81 @@
-<h1>{{title}}</h1>
-<div igxLayout igxLayoutDir="columns" igxLayoutItemAlign="center">
-    <div>
-        <p>Open terminal and do:</p>
-<pre class="code-snippet left-align">
-$ npm i -g igniteui-cli
-$ ig
-</pre>
-        <p>Then simply follow the CLI guidance and choose Angular. That's it!</p>
-        <h3>Prerequisites:</h3>
-        <p class="small-links"><a href="https://nodejs.org" alt="Node.js">Node.js</a> v8.9.4+</p>
-        <p class="small-links"><a href="https://www.npmjs.com/" alt="npmjs">npm</a> v5+</p>
-    </div>
-    <div id="linksContainer">
-        <h3 class="links"><a target="_blank" rel="noopener" href="https://github.com/IgniteUI/ignite-ui-cli">Ignite UI CLI </a></h3>
-        <h3 class="links"><a target="_blank" rel="noopener" href="https://www.infragistics.com/products/ignite-ui-angular">Component Demos Online</a></h3>
-    </div>
-</div>
\ No newline at end of file
+<div class="home">
+    <section class="header">
+        <h2 class="title">
+            Welcome to Ignite UI for Angular!
+        </h2>
+        <p class="description">
+            Browse the left menu to see the complete set of samples, including source code, for each control. You’ll see the full array
+            of capabilities in Ignite UI for Angular, all the different features for the controls, how to implement those
```

Comment:
"how to implement those features" is not shown either. One should actually open the project in IDE to see that.